.github/labeler.yml

```diff
@@ -61,3 +61,8 @@
       - CODEOWNERS
       - MANIFEST.in
       - noxfile.py
+
+"part:actor":
+  - changed-files:
+    - any-glob-to-any-file:
+      - "src/frequenz/actor/dispatch/**"
```

Comment:
Will fix in the next pr

Reply:
Maybe not? :P

Reply:
On second glance, this involved WAY more files than I anticipated.

Reply:
Hmm but the client has the structure/namespace `frequenz.client.dispatch`

Reply:
> On second glance, this involved WAY more files than I anticipated.

Why?

> Hmm but the client has the structure/namespace `frequenz.client.dispatch`

Yes, and it is in a repo called `frequenz-client-dispatch`. I mean, we can just name this repo `frequenz-actor-dispatch` and have only the actor as we planned originally, I just thought it might have been a good opportunity to try this other scheme, by mixing the actor and high-level interface in one repo, and keeping the actor as an implementation detail, as we do in the SDK, where users never need to deal with the actual actors used behind the scenes, and just use higher-level functions and objects that interact with the actors for them.

Reply:
Oh, I see the diff now, so you actually used the `actor` template to create this repo, not the `lib` template, right?

Reply:
Yes, I used `actor`

>>On second glance, this involved WAY more files than I anticipated.
>
>Why?

Because the name/path `frequenz-actor-dispatch` was all over the code, dependencies, tests, etc

> I just thought it might have been a good opportunity to try this other scheme

Ah, alright. Sure, we can try.